client/tests/__mocks__/GraphQLMocks.js

```diff
@@ -0,0 +1,464 @@
+                                'VoiceOver for macOS 10.14 with Firefox 50.43',
+                            at: {
+                                id: '3',
+                                name: 'VoiceOver for macOS',
+                                __typename: 'At'
+                            },
+                            browser: {
+                                id: '1',
+                                name: 'Firefox',
+                                __typename: 'Browser'
+                            },
+                            atVersion: '10.14',
+                            browserVersion: '50.43',
+                            __typename: 'TestPlanTarget'
+                        },
+                        testPlanVersion: {
+                            id: '6',
+                            title: 'Editor Menubar Example',
+                            gitSha: '4ca7842ea7777b668546e74c9b5ed5b09696d927',
+                            gitMessage:
+                                'Revert "Generated tests and review pages output improvements (#441)" (#449)',
+                            directory: 'menubar-editor',
+                            testCount: 40,
+                            __typename: 'TestPlanVersion'
+                        },
+                        draftTestPlanRuns: [],
+                        __typename: 'TestPlanReport'
+                    },
+                    {
+                        id: '11',
+                        status: 'DRAFT',
+                        testPlanTarget: {
+                            id: '8',
+                            title: 'JAWS Ok with Chrome 1000',
+                            at: {
+                                id: '1',
+                                name: 'JAWS',
+                                __typename: 'At'
+                            },
+                            browser: {
+                                id: '2',
+                                name: 'Chrome',
+                                __typename: 'Browser'
+                            },
+                            atVersion: 'Ok',
+                            browserVersion: '1000',
+                            __typename: 'TestPlanTarget'
+                        },
+                        testPlanVersion: {
+                            id: '2',
+                            title: null,
+                            gitSha: '4ca7842ea7777b668546e74c9b5ed5b09696d927',
+                            gitMessage:
+                                'Revert "Generated tests and review pages output improvements (#441)" (#449)',
+                            directory: 'checkbox-tri-state',
+                            testCount: 24,
+                            __typename: 'TestPlanVersion'
+                        },
+                        draftTestPlanRuns: [],
+                        __typename: 'TestPlanReport'
+                    }
+                ]
+            }
+        }
+    }
+];
```

Comment:
I think this is a valid approach - although part of me wonders if there's a way to use the dbCleaner functionality and the test database. That would allow us to test code that uses mutations and it would allow us to test the backend and frontend simultaneously. Maybe that's too ambitious but I'm curious what you think.

Reply:
Well I admittedly didn't look too long into it but best practice here indicates the use of the `MockedProvider` which could certainly become difficult to maintain the more this grows which is what this is for, or some automatic mocking HOC around the `ApolloProvider`.

> although part of me wonders if there's a way to use the dbCleaner functionality and the test database

Making use of the dbCleaner could certainly help with creating some more dynamic and robust tests. It'd definitely be ambitious as you said and will take some time ha, but it would be beneficial. We could look into it.

Reply:
Ok, perhaps it's an option ... but no need to hold this PR up over it 😜